engine/src/main/java/org/entando/entando/web/page/model/PageRequest.java

```diff
@@ -17,27 +17,29 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import org.hibernate.validator.constraints.NotBlank;
+
+import javax.validation.constraints.NotNull;
+//import org.hibernate.validator.constraints.NotBlank;
 
 /**
  *
  * @author paddeo
  */
 public class PageRequest {
 
-    @NotBlank(message = "page.code.NotBlank")
+    @NotNull(message = "page.code.NotBlank")

```

Comment:
I prefer using the javax.validation classes too, but I think here we need also a `@Size` constraint.

Reply:
I think that we have to keep the same constraints that are actually used in the action. If you look at the `PageAction-validation.xml` (inside admin-console module) you can see that the size can be between 1 and 30 and there is also a regex validation.